frontend/src/containers/404/not-found.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Lottie from 'react-lottie'
```

Comment:
This is a third imports, to separate it

// React imports
import React from 'react'

// Third imports
import Lottie from 'react-lottie'

// Project imports
import notFound from '../../static/lottie/404.json'